src/other/helloworld/demo/bonjour.js

```diff
@@ -17,6 +17,7 @@ var nomComposant = "wb-bonjour",
 	selecteur = "." + nomComposant,
 	initEvent = "wb-init" + selecteur,
 	$document = wb.doc,
+	defaut = {},
```

Comment:
Speaking of... it's nitpicking, not knit picking. :-) ;-)
https://en.wikipedia.org/wiki/Nitpicking